### PR TITLE
Refinements to object descriptions (L:Z)

### PR DESCRIPTION
**Pull request by floydtree**
#### Description of changes:
1. Updating descriptions of objects (L-Z), non-breaking.

### CODE REVIEW

**Review by rroupski — DISMISSED** [review]

**Review by jasonbreimer — DISMISSED** [review]

File: objects/service.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "caption": "Service",
-  "name": "service",
-  "description": "The information pertaining to the requested service.",
+  "description": "Descrbies characteristics of a service, <code> e.g. AWS EC2. </code>",
```
Comment (Aniak5) [inline objects/service.json:3]: Typo in Describes!

File: objects/tls_extension.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
-  "description": "The Transport Layer Security (TLS) extension object.",
   "caption": "TLS Extension",
-  "name": "tls_extension",
+  "description": "Descrbies additional attributes that extend the base Transport Layer Security (TLS) object.",
```
Comment (Aniak5) [inline objects/tls_extension.json:3]: Typo in describes

File: objects/san.json [hunk 3]
```diff
@@ -1,8 +1,8 @@
 {
   "caption": "Subject Alternative Name",
-  "name": "san",
+  "description": "Descrbies a Subject Alternative name (SAN) secured by a digital certificate",
```
Comment (Aniak5) [inline objects/san.json:3]: Typo in describes

File: objects/service.json (hunk 1 above)
Comment (floydtree) [reply objects/service.json:3]: good catch!

File: objects/san.json (hunk 3 above)
Comment (floydtree) [reply objects/san.json:3]: lol, quite a pattern

**Review by Aniak5 — APPROVED** [review]
LGTM!

**Review by rroupski — APPROVED** [review]

**Review by rroupski — CHANGES_REQUESTED** [review]
We should keep the original wording, _The Foo object describes/contains/etc_ ... It was recommended/used by the Symantec tech writer, who wrote the original descriptions for ICD schema. 

BTW, all changes that I made in `_ to K` commit use that style, and the object name is always in Title Case. For example: _The Email object describes the email metadata such as sender, ..._

**Comment by floydtree** [comment]
> We should keep the original wording, _The Foo object describes/contains/etc_ ... It was recommended/used by the Symantec tech writer, who wrote the original descriptions for ICD schema.
> 
> BTW, all changes that I made in `_ to K` commit use that style, and the object name is always in Title Case. For example: _The Email object describes the email metadata such as sender, ..._

 I think it's adds unnecessary redundancy to the descriptions, but considering yours are already merged I'll change.

**Review by rroupski — APPROVED** [review]

**Review by jasonbreimer — APPROVED** [review]

**Review by Aniak5 — APPROVED** [review]
LGTM

**Pull request merged**